trickster/routing/input.py

```diff
@@ -115,14 +124,16 @@ def __init__(
         method: str,
         headers: Dict[str, str] = None,
         form: Dict[str, str] = None,
-        cookies: Dict[str, str] = None
+        cookies: Dict[str, str] = None,
+        body: str = None
```

Comment:
```suggestion
        body: str = ''
```
that helper
```python
foo = None
self.foo = foo or {}
```
is used for mutable default, which string is not. 

But you may have some reason to use None instead of empty string which I am not seeing.

Reply:
yeah,  I know... maybe I was trying to be consistent with the others arguments, no matter what is mutable or not. But I'll go on your way